examples/base/README.md

```diff
@@ -0,0 +1,26 @@
+# RainCatcher BaseModule
+
+This is a template for modules in the RainCatcher project, it contains the default dotfiles and other configuration files for the development tooling and the directory structure spected from individual modules.
+
+RainCatcher code is written in [TypeScript](http://typescriptlang.org), uses [Mocha](https://mochajs.org/) for unit tests and [Istanbul](https://istanbul.js.org/) for test coverage checks.
+
+For an explanation of the expected internals of see the [main README](../../README.md#package-structure)
```

Comment:
"...expected internals of see the main README", misplaced of? Should this be `For an explanation of the expected internals, see the main README`